### PR TITLE
Add EditUserDescriptorTest

### PR DESCRIPTION
**Pull request by helloImHai**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import static seedu.address.testutil.TypicalEditUserDescriptor.AMY_DESC;
+import static seedu.address.testutil.TypicalEditUserDescriptor.BOB_DESC;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.EditUserDescriptorBuilder;
+
+public class EditUserDescriptorTest {
+    @Test
+    public void isAnyFieldMissing_noFieldEdited_returnsTrue() {
```
Comment (duongphammmm) [inline src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java]: should this be `isAnyFieldEdited_noFieldEdited_returnsFalse()` ?

File: src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    /**
+     * Sets the {@code From} of the {@code EditUserDescriptor} that we are building.
+     */
+    public EditUserDescriptorBuilder withFrom(String from) {
+        descriptor.setFrom(new Time(from));
+        return this;
+    }
+
+    /**
+     * Sets the {@code From} of the {@code EditUserDescriptor} that we are building.
```
Comment (duongphammmm) [inline src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java]:
```suggestion
     * Sets the {@code To} of the {@code EditUserDescriptor} that we are building.
```

Comment (helloImHai) [reply src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java]: Nice catch

File: src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java (hunk 1 above)
Comment (helloImHai) [reply src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java]: Yes

Comment (duongphammmm) [inline src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java:26]:
```suggestion
    public void isAnyFieldEdited_noFieldEdited_returnsFalse() {
```

File: src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java (hunk 2 above)
Comment (duongphammmm) [inline src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java:130]:
```suggestion
     * Sets the {@code To} of the {@code EditUserDescriptor} that we are building.
```

File: src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java (hunk 1 above)
Comment (duongphammmm) [inline src/test/java/seedu/address/logic/commands/edit/EditUserDescriptorTest.java]:
```suggestion
    public void isAnyFieldEdited_noFieldEdited_returnsFalse() {
```

File: src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java (hunk 2 above)
Comment (duongphammmm) [inline src/test/java/seedu/address/testutil/EditUserDescriptorBuilder.java]:
```suggestion
     * Sets the {@code To} of the {@code EditUserDescriptor} that we are building.
```

**Pull request merged**